tracpro/baseline/forms.py

```diff
@@ -44,10 +45,82 @@ def clean(self, *args, **kwargs):
 
         baseline_question = cleaned_data.get("baseline_question")
         follow_up_question = cleaned_data.get("follow_up_question")
+        if baseline_question and follow_up_question and baseline_question == follow_up_question:
+            raise forms.ValidationError(
+                "Baseline question and follow up question should be different."
+            )
 
+        return cleaned_data
+
+
+class SpoofDataForm(forms.Form):
+    """
+    Form to create spoofed poll data
+    """
+    contacts = forms.ModelMultipleChoiceField(queryset=Contact.objects.all(),
+                                              help_text=_("Select contacts for this set of spoofed data."))
+    start_date = forms.DateField(help_text=_(
+        "Baseline poll data will be submitted on this date. " +
+        "Follow up data will start on this date."))
+    end_date = forms.DateField(help_text=_("Follow up data will end on this date."))
+    baseline_question = forms.ModelChoiceField(queryset=Question.objects.all(),
+                                               help_text=_(
+                                               "Select a baseline question which " +
+                                               "will have numeric answers only."))
+    follow_up_question = forms.ModelChoiceField(queryset=Question.objects.all(),
+                                                help_text=_(
+                                                "Select a follow up question which " +
+                                                "will have numeric answers only."))
+    baseline_minimum = forms.IntegerField(
+        help_text=_("A baseline answer will be created for each contact within the minimum/maximum range."))
+    baseline_maximum = forms.IntegerField(
+        help_text=_("A baseline answer will be created for each contact within the minimum/maximum range."))
+    follow_up_minimum = forms.IntegerField(
+        help_text=_("Follow up answers will be created for each contact within the minimum/maximum range."))
+    follow_up_maximum = forms.IntegerField(
+        help_text=_("Follow up answers will be created for each contact within the minimum/maximum range."))
+
+    def __init__(self, *args, **kwargs):
+        org = kwargs.pop('org')
+
+        super(SpoofDataForm, self).__init__(*args, **kwargs)
+
+        if org:
```

Comment:
Looks like this argument comes from `request.org`. Is it possible to get to this page without an org? If not, the conditional can be removed.